src/components/layout/CyVerseAppBar.js

```diff
@@ -69,6 +68,9 @@ import UserMenu from "./UserMenu";
 // hidden in xsDown
 const GlobalSearchField = dynamic(() => import("../search/GlobalSearchField"));
 
+// only new users
+const Joyride = dynamic(() => import("react-joyride"));
```

Comment:
Looks like next is smart enough to understand importing react-joyride twice, both dynamically, and only making one chunk/loading once. So that's cool.